### PR TITLE
autodetect efi directory

### PR DESCRIPTION
**Pull request by karmab**

### CODE REVIEW

File: prepare-efi.sh [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 set -euxo pipefail
 
-OS=${1:-centos}
 DEST=${2:-/tmp/esp.img}
```
Comment (LorbusChris) [inline prepare-efi.sh:5]: This would become the new first arg

File: Dockerfile.fcos [hunk 2]
```diff
@@ -4,7 +4,7 @@ FROM fedora:37 AS builder
 WORKDIR /tmp
 
 COPY prepare-efi.sh /bin/
-RUN prepare-efi.sh fedora
```
Comment (LorbusChris) [inline Dockerfile.fcos:7]: Dockerfile.scos needs the analogue change

**Review by LorbusChris — APPROVED** [review]

**Review by elfosardo — CHANGES_REQUESTED** [review]
was this tested? does this need to be changed also upstream in metal3?

**Pull request closed without merging**